homeassistant/components/hyundai_kia_connect/sensor.py

```diff
@@ -0,0 +1,117 @@
+"""Sensor for Hyundai / Kia Connect integration."""
```

Comment:
Let's remove the sensor platform for now and get the integration merged with a single platform (binary sensor) first.

Additional platforms can be added in follow up PRs